admin/plugins.php

```diff
@@ -0,0 +1,411 @@
+<?php
+/**
+ * Admin settings helper functions.
+ *
+ * @package performance-lab
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Gets plugin info from plugin slug.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $plugin_slug The string identifier for the plugin in questions slug.
+ *
+ * @return array Array of plugin data, or empty if none/error.
+ */
+function perflab_get_plugin_info( string $plugin_slug ) {
```

Comment:
Maybe rename to `perflab_query_plugin_info()` to clarify the remote request nature (and thus cost) of this function?